add christian hoerauf

fonzerelly opened the pull request:
I would like to register for JS CraftCamp.    
- [x ] My Pull Request contains a json file `$firstname_$lastname.json` or `$nickname.json`    
- [x ] The json file follows the template at https://github.com/jscraftcamp/website/blob/master/participants/_template.json and contains the mandatory fields `name`, `tags`, `when`, `what_is_my_connection_to_javascript` and `what_can_i_contribute`    
- [x ] If I can't attend, I will either send another Pull Request removing my json file or an E-Mail with the subject 'UNREGISTER' to team@jscraftcamp.org

davelosert commented on participants/christian_hoerauf.json:
```diff
@@ -0,0 +1,29 @@
+  "email": "christian.hoerauf@consorsbank.de",
+  "what_is_my_connection_to_javascript": "After being frustrated with C++, I got started into JavaScript and Functional Programming. I love TDD and looking forward for the next technology to turn my head around for 180 degrees.",
+  "what_can_i_contribute": "I have a lot experiance with Angular1 and am currently introducing Angular2 to my company. I also know jasmine very well and developed mocking.js to simplify the generation and usage of mockdata.",
+  "tshirt": "M-XL/",
```
There is an Slah too much at the end here - if you make it `M-XL` you should be fine :)

davelosert commented on participants/christian_hoerauf.json:
```diff
@@ -0,0 +1,29 @@
+  "tshirt": "M-XL/",
+  "urls": {
+    "photo": "https://drive.google.com/open?id=0B3-B1iwb9igGWFFsa1RFTGstUzQ",
+    "homepage": "",
```
Please simply remove the field instead of leaving it empty, then the tests will passt.

fonzerelly commented:
Of course this has to happen me. But what would you expect, when you have to do such a registration, while your children jump around you: "Dad, can we play football?" "Daaad, can we go swimming?"...

Thax for remarking my mistakes. Has there been some kind of automated Test I should /could have run?
I have to wait until this evening - at work, my access to github is limited.

davelosert commented:
@fonzerelly : haha oh no. I hope you still had the time to play with you children.
You can actually run the tests by executing `yarn test` in the Project Directory.
You can also click on "Details" on the travigs link under this PR to see what tests failed on the server.

It's important that the tests are passing as our homepage directly reads the data and expects a certain format. In the worst case, a merge with failing tests would make our Homepage-Build fail :smile: 

No rush with the fixes though - there are still enough open spots and since you already put a PR it will be the first to be merged as soon as it's fixed.

fonzerelly commented:
Hi dave, looks like this bugfix did it. By the way:
Each time when I clone your repo, I always get some strange git error:
 _git clone https://github.com/jscraftcamp/website.git website2
Cloning into 'website2'...
remote: Counting objects: 4022, done.
remote: Compressing objects: 100% (63/63), done.
remote: Total 4022 (delta 43), reused 45 (delta 14), pack-reused 3945
Receiving objects: 100% (4022/4022), 21.04 MiB | 345.00 KiB/s, done.
Resolving deltas: 100% (2456/2456), done.
error: unable to create file src/img/pr_screenshots/Screenshot from 2016-08-31 17:55:31.png: Invalid argument
error: unable to create file src/img/pr_screenshots/Screenshot from 2016-08-31 18:12:31.png: Invalid argument
fatal: unable to checkout working tree
warning: Clone succeeded, but checkout failed.
You can inspect what was checked out with 'git status'
and retry the checkout with 'git checkout -f HEAD'_


It also leads to that weird default git status:
_On branch master
Your branch is up-to-date with 'origin/master'.
Changes to be committed:
  (use "git reset HEAD <file>..." to unstage)

	deleted:    .editorconfig
	deleted:    .github/PULL_REQUEST_TEMPLATE.md
	deleted:    .gitignore
	deleted:    .travis.yml
	deleted:    CNAME
	deleted:    README.md
	deleted:    logo.svg
	deleted:    package.json
	deleted:    participants/Marcel_Fiebelkorn.json
	deleted:    participants/_template.json
	deleted:    participants/ana_carraca.json
	deleted:    participants/andreas_harter.json
	deleted:    participants/andreas_jaegle.json
	deleted:    participants/andreas_neumann.json
	deleted:    participants/angela_beltran.json
	deleted:    participants/axel_hodler.json
	deleted:    participants/brigitte_jellinek.json
	deleted:    participants/carlo_jeske.json
	deleted:    participants/carsten_lamm.json
	deleted:    participants/christian_waldmann.json
	deleted:    participants/daniel_hilpoltsteiner.json
	deleted:    participants/david_losert.json
	deleted:    participants/dimitry_leonov.json
	deleted:    participants/fabian_matern.json
	deleted:    participants/florian_loretan.json
	deleted:    participants/frank_blendinger.json
	deleted:    participants/frank_mueller.json
	deleted:    participants/invited/.keep
	deleted:    participants/jesus_real.json
	deleted:    participants/joern_bernhardt.json
	deleted:    participants/johann_sonntagbauer.json
	deleted:    participants/jonathan_haeberle.json
	deleted:    participants/jonathan_juhas.json
	deleted:    participants/lars_gregori.json
	deleted:    participants/mario_volke.json
	deleted:    participants/markus_knittig.json
	deleted:    participants/markus_westphal-furuya.json
	deleted:    participants/martin_lechner.json
	deleted:    participants/michael_czechowski.json
	deleted:    participants/michael_jaser.json
	deleted:    participants/michael_ohr.json
	deleted:    participants/michael_zoidl.json
	deleted:    participants/nikolas_odonnell.json
	deleted:    participants/paul_torka.json
	deleted:    participants/radu_baranga.json
	deleted:    participants/ruben_straube.json
	deleted:    participants/stefan_rohe.json
	deleted:    participants/thanh_le.json
	deleted:    participants/thomas_mayrhofer.json
	deleted:    participants/thomas_traenkler.json
	deleted:    participants/walter_kammergruber.json
	deleted:    participants/yoyo_gruber.json
	deleted:    participants/zied_ghalleb.json
	deleted:    scripts/concat-json.js
	deleted:    scripts/deploy-to-ghpages.sh
	deleted:    scripts/encrypt-github-token-for-travis.sh
	deleted:    scripts/for-travis/build.sh
	deleted:    src/404.md
	deleted:    src/_data.json
	deleted:    src/_footer.ejs
	deleted:    src/_layout.ejs
	deleted:    src/_navigation.ejs
	deleted:    src/css/main.css
	deleted:    src/css/materialize.min.css
	deleted:    src/css/prism-customized.css
	deleted:    src/fonts/font-awesome/css/font-awesome.css
	deleted:    src/fonts/font-awesome/css/font-awesome.min.css
	deleted:    src/fonts/font-awesome/fonts/FontAwesome.otf
	deleted:    src/fonts/font-awesome/fonts/fontawesome-webfont.eot
	deleted:    src/fonts/font-awesome/fonts/fontawesome-webfont.svg
	deleted:    src/fonts/font-awesome/fonts/fontawesome-webfont.ttf
	deleted:    src/fonts/font-awesome/fonts/fontawesome-webfont.woff
	deleted:    src/fonts/material-design-icons/LICENSE.txt
	deleted:    src/fonts/material-design-icons/Material-Design-Icons.eot
	deleted:    src/fonts/material-design-icons/Material-Design-Icons.svg
	deleted:    src/fonts/material-design-icons/Material-Design-Icons.ttf
	deleted:    src/fonts/material-design-icons/Material-Design-Icons.woff
	deleted:    src/fonts/material-design-icons/Material-Design-Icons.woff2
	deleted:    src/fonts/roboto/Roboto-Bold.eot
	deleted:    src/fonts/roboto/Roboto-Bold.ttf
	deleted:    src/fonts/roboto/Roboto-Bold.woff
	deleted:    src/fonts/roboto/Roboto-Bold.woff2
	deleted:    src/fonts/roboto/Roboto-Light.eot
	deleted:    src/fonts/roboto/Roboto-Light.ttf
	deleted:    src/fonts/roboto/Roboto-Light.woff
	deleted:    src/fonts/roboto/Roboto-Light.woff2
	deleted:    src/fonts/roboto/Roboto-Medium.eot
	deleted:    src/fonts/roboto/Roboto-Medium.ttf
	deleted:    src/fonts/roboto/Roboto-Medium.woff
	deleted:    src/fonts/roboto/Roboto-Medium.woff2
	deleted:    src/fonts/roboto/Roboto-Regular.eot
	deleted:    src/fonts/roboto/Roboto-Regular.ttf
	deleted:    src/fonts/roboto/Roboto-Regular.woff
	deleted:    src/fonts/roboto/Roboto-Regular.woff2
	deleted:    src/fonts/roboto/Roboto-Thin.eot
	deleted:    src/fonts/roboto/Roboto-Thin.ttf
	deleted:    src/fonts/roboto/Roboto-Thin.woff
	deleted:    src/fonts/roboto/Roboto-Thin.woff2
	deleted:    src/help_pull_request.ejs
	deleted:    src/img/atoss_logo.png
	deleted:    src/img/cc_logo.png
	deleted:    src/img/event_photos/CuOyZ_kWYAARNY0.jpg
	deleted:    src/img/event_photos/CuPNlg6XEAA8HtJ.jpg
	deleted:    src/img/event_photos/CuaYSc3WYAAdRRV.jpg
	deleted:    src/img/event_photos/CuaYwEuXgAEEkzT.jpg
	deleted:    src/img/event_photos/CuaZHUSW8AAOLs3.jpg
	deleted:    src/img/event_photos/IMG_5352.jpg
	deleted:    src/img/event_photos/IMG_5374.jpg
	deleted:    src/img/event_photos/IMG_5375.jpg
	deleted:    src/img/event_photos/IMG_5377.jpg
	deleted:    src/img/event_photos/IMG_5381.jpg
	deleted:    src/img/event_photos/IMG_5394.jpg
	deleted:    src/img/event_photos/IMG_5395.jpg
	deleted:    src/img/event_photos/IMG_5404.jpg
	deleted:    src/img/event_photos/IMG_5413.jpg
	deleted:    src/img/event_photos/IMG_5418.jpg
	deleted:    src/img/event_photos/IMG_5419.jpg
	deleted:    src/img/event_photos/IMG_5422.jpg
	deleted:    src/img/gf_logo.png
	deleted:    src/img/innoq_logo.png
	deleted:    src/img/jscc-logo-without-text.svg
	deleted:    src/img/logo.png
	deleted:    src/img/nophoto.png
	deleted:    src/img/pr_screenshots/Screenshot from 2016-08-31 17:55:31.png
	deleted:    src/img/pr_screenshots/Screenshot from 2016-08-31 18:12:31.png
	deleted:    src/img/pr_screenshots/clone.png
	deleted:    src/img/pr_screenshots/compare.png
	deleted:    src/img/pr_screenshots/compare_forks.png
	deleted:    src/img/pr_screenshots/fill_out_comment.png
	deleted:    src/img/pr_screenshots/fork.png
	deleted:    src/img/pr_screenshots/open_pr.png
	deleted:    src/img/pr_screenshots/travis_checking.png
	deleted:    src/img/pr_screenshots/travis_details.png
	deleted:    src/img/pr_screenshots/travis_fail.png
	deleted:    src/img/pr_screenshots/travis_success.png
	deleted:    src/img/scandio_logo_google.jpg
	deleted:    src/img/simplelabs-logo.png
	deleted:    src/img/sponsor-2017/codecentric.png
	deleted:    src/img/sponsor-2017/holidaycheck.jpg
	deleted:    src/img/sponsor-2017/mercateo.jpg
	deleted:    src/img/sponsor-2017/peerigon.png
	deleted:    src/img/sponsor-2017/sinnerschrader.png
	deleted:    src/img/sponsor-2017/trustyou-logo.png
	deleted:    src/img/sponsor-2017/wunder.png
	deleted:    src/img/theteam/david.jpg
	deleted:    src/img/theteam/marco.jpg
	deleted:    src/img/theteam/megan.jpg
	deleted:    src/img/theteam/robert.png
	deleted:    src/img/theteam/stefanie.png
	deleted:    src/img/theteam/wolfram.png
	deleted:    src/img/tng_logo.png
	deleted:    src/img/vi_logo.jpeg
	deleted:    src/imprint.md
	deleted:    src/index.ejs
	deleted:    src/info2016.ejs
	deleted:    src/js/handlebars.js
	deleted:    src/js/index.js
	deleted:    src/js/jquery-2.1.1.min.js
	deleted:    src/js/materialize.min.js
	deleted:    src/js/mc-validate.js
	deleted:    src/js/prism-line-highlight.js
	deleted:    src/js/prism.js
	deleted:    src/js/tshirt-sizes.js
	deleted:    src/participants.ejs
	deleted:    src/participants2016.ejs
	deleted:    src/participants2016.json
	deleted:    src/photos2016.ejs
	deleted:    src/registration.ejs
	deleted:    src/sponsoring.ejs
	deleted:    src/theteam.ejs
	deleted:    src/values.ejs
	deleted:    test/participants.js
	deleted:    test/tshirt-sizes.js
	deleted:    tshirt-design/JS CraftCamp.svg
	deleted:    tshirt-design/JSCraftCampVorlage.eps
	deleted:    tshirt-design/JSCraftCampVorlage.odg
	deleted:    tshirt-design/frontside-participants.svg
	deleted:    tshirt-design/one-color-logos/atoss.svg
	deleted:    tshirt-design/one-color-logos/codecentric.svg
	deleted:    tshirt-design/one-color-logos/gutefrage.svg
	deleted:    tshirt-design/one-color-logos/innoq.svg
	deleted:    tshirt-design/one-color-logos/scandio.svg
	deleted:    tshirt-design/one-color-logos/simplabs.svg
	deleted:    tshirt-design/one-color-logos/tng.svg
	deleted:    tshirt-design/one-color-logos/vi.svg
	deleted:    yarn.lock

Untracked files:
  (use "git add <file>..." to include in what will be committed)

	.editorconfig
	.github/
	.gitignore
	.travis.yml
	CNAME
	README.md
	logo.svg
	package.json
	participants/
	scripts/
	src/
	test/
	tshirt-design/
	yarn.lock_


Am I using the wrong git version?
git version
git version 2.11.0.windows.3

Fornuatelly, I knew how to commit only my file ;)

davelosert commented:
@fonzerelly : Hum, maybe becuase of the ":" in the name - could be that it doesn't play well with windows.
I changed that in another PR.

Your Tests are passing now and everything is fine! :+1: 
I am looking forward on seeing you at JSCC17! :smile:
